Formula/dvc.rb

```diff
@@ -22,6 +22,7 @@ class Dvc < Formula
   depends_on "apache-arrow"
   depends_on "libgit2"
   depends_on "libpython-tabulate"
+  depends_on "numpy"
```

Comment:
I think `numpy` isn't used right now since we don't build `pyarrow`. It should be part of that dependency tree:
```
├── dvc-hdfs==2.19.0 (2.19.0)
│   ├── dvc (2.25.0, cyclic)
│   └── fsspec[arrow] (2022.8.2)
│       └── pyarrow>=1 (9.0.0)
│           └── numpy>=1.16.6 (1.23.3)
```